steamos-waydroid-installer.sh

```diff
@@ -397,7 +398,30 @@ else
 		elif [ "$Choice" == "TV" ]
 		then
 			echo Android TV chosen!
-			exit
+			echo Initializing Waydroid
+			echo -e "$current_password\n" | sudo -S waydroid init
+			echo Downloading Android TV image
+            echo -e "$current_password\n" | sudo -S curl -o ~/waydroid/images/androidtv.zip $ANDROID_TV_IMG -L
+			echo Extracting Archive
+			echo -e "$current_password\n" | sudo -S unzip -o ~/waydroid/images/androidtv -d ~/waydroid/images
+			echo -e "$current_password\n" | sudo -S rm ~/waydroid/images/androidtv.zip
+			echo Reinitializing Waydroid
+			echo -e "$current_password\n" | sudo -S waydroid init -f
+		    if zenity --question --text="Install MicroG and Aurora Store"; then
+			 echo -e "$current_password\n" | sudo -S mkdir ~/waydroid/apks
+			 echo -e "$current_password\n" | sudo -S curl -L -o ~/waydroid/apks/com.google.android.gms-244735012.apk https://github.com/microg/GmsCore/releases/download/v0.3.6.244735/com.google.android.gms-244735012.apk
+			 echo -e "$current_password\n" | sudo -S curl -L -o ~/waydroid/apks/AuroraStore-4.6.4.apk https://auroraoss.com/downloads/AuroraStore/Release/AuroraStore-4.6.4.apk
+			 konsole -e bash /home/deck/Android_Waydroid/Android_Waydroid_Cage.sh &
+			 sleep 30
+			 waydroid app install ~/waydroid/apks/com.google.android.gms-244735012.apk
+			 waydroid app install ~/waydroid/apks/AuroraStore-4.6.4.apk
+			 sleep 40
+		     echo "$current_password\n" | sudo -S waydroid container stop
+
+            else
+              echo OK!
+            fi
```

Comment:
should also be tabs vs spaces

```suggestion
			else
				echo OK!
			fi
```